src/expr/macro/src/lib.rs

```diff
@@ -24,6 +24,274 @@ mod parse;
 mod types;
 mod utils;
 
+/// Defining expression from a Rust Function.
```

Comment:
```suggestion
/// Defining expression from a Rust Function. [Online version of this doc](https://risingwavelabs.github.io/risingwave/risingwave_expr_macro/attr.function.html).
```
Since it's quite long 😄

Reply:
Wow I didn't even notice this online doc.